modules/about-project-creation.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/configuring-project-creation.adoc
+
+[id="about-project-creation_{context}"]
+= About project creation
+
+The {product-title} API server automatically provisions new projects based on
+the project template that is identified by the `projectRequestTemplate`
+parameter in the cluster's Project configuration resource. If the parameter is
```

Comment:
Should Project be lowercase? [1]

[1] https://github.com/openshift/openshift-docs/blob/enterprise-4.1/contributing_to_docs/term_glossary.adoc#project

I see elsewhere in this PR that it's capitalized and not sure if it should be lower everywhere.

Reply:
Had instances of "Project configuration resource" as uppercase to match how it appears in the web console, but have made lowercase now. Will follow-up w/ the team though about how we're referring to configuration resources in general as I'm seeing many different stylings across the docs.